pwndbg/commands/aslr.py

```diff
@@ -32,12 +76,17 @@
 @pwndbg.commands.ArgparsedCommand(parser, category=CommandCategory.LINUX)
 def aslr(state=None) -> None:
     if state:
-        gdb.execute(f"set disable-randomization {options[state]}", from_tty=False, to_string=True)
+        if pwndbg.dbg.is_gdblib_available():
+            gdb.execute(
+                f"set disable-randomization {options[state]}", from_tty=False, to_string=True
+            )
 
-        if pwndbg.gdblib.proc.alive:
-            print("Change will take effect when the process restarts")
+            if pwndbg.aglib.proc.alive:
+                print("Change will take effect when the process restarts")
+        else:
+            print("Could not change ASLR on LLDB")
```

Comment:
do we want to expose this command on lldb if it doesn't work?

Reply:
We should probably implement this differently, through a decorator like `@SupportedOnlyOn(GDB|LLDB)`?

Reply:
todo: `settings set target.disable-aslr false`